### PR TITLE
KAH-21

### PR DESCRIPTION
**Pull request by DavidIzaac**
Remove the :projectId/add/:mapId endpoint from the project route

### CODE REVIEW

File: console/src/app/maps/map-detail/map-properties/map-properties.component.ts [hunk 1]
```diff
@@ -88,7 +88,7 @@ export class MapPropertiesComponent implements OnInit, OnDestroy {
   }
 
   onChangeProject() {
-    let mapObj = Object.assign({}, this.map, { project: {id:this.selectedProject} });
+    let mapObj = Object.assign({}, this.map, { project: this.selectedProject});
```
Comment (arispen) [inline console/src/app/maps/map-detail/map-properties/map-properties.component.ts]: why this change is in this pull request?
what it has to do with this issue?

Comment (ChBernat) [reply console/src/app/maps/map-detail/map-properties/map-properties.component.ts]: I brought it back.

Comment (DavidIzaac) [reply console/src/app/maps/map-detail/map-properties/map-properties.component.ts]: This is was a bug that I spoke to @theflexbird that we cannot change project of a specific map. Because it was a little change, I didn't created a new task. @ChBernat Please keep like it was.
